dynamic_programming/shortest_common_supersequence.cpp

```diff
@@ -0,0 +1,171 @@
+/**
+ * @shortest_common_supersequence.cpp
+ * @brief SCS is a string Z which is the shortest supersequence of strings X and Y (may not be continuous in Z, but order is maintained).
+ * @details
+ * Details for [Shortest Common Supersequence](https://en.wikipedia.org/wiki/Shortest_common_supersequence_problem)
```

Comment:
Please remove this line, it's not needed.

Reply:
removed 
```
* @detail
* Details for ...
```
and added in 
```
* @see more on [SCS](...)
```

Is this right?